contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -209,7 +211,7 @@ contract UniswapAnchoredView is UniswapConfig {
      * @param message The data that was presumably signed
      * @param signature The fingerprint of the data + private key
      */
-    function invalidateReporter(bytes memory message, bytes memory signature) external {
+    function invalidateReporter(bytes calldata message, bytes calldata signature) external {
```

Comment:
I think we should switch to `solc 0.6.10` pragma. Note I already switched the Dockerfile / CI solc so that it would compile this line without complaint

Reply:
I can change to `pragma solidity ^0.6.10;` in all related files.